internal/inspectimage/bom_display.go

```diff
@@ -36,8 +36,9 @@ func displayBOM(bom []lifecycle.BOMEntry) []BOMEntryDisplay {
 
 			Buildpack: dist.BuildpackRef{
 				BuildpackInfo: dist.BuildpackInfo{
-					ID:      entry.Buildpack.ID,
-					Version: entry.Buildpack.Version,
+					ID:       entry.Buildpack.ID,
+					Version:  entry.Buildpack.Version,
+					Homepage: entry.Buildpack.Homepage,
```

Comment:
I'm still not able to observe this behavior on the command-line. I need help triggering this behavior.

Reply:
Ya had trouble doing this during acceptance as well. Unfortunately seems like this is impossible right now because of this line in the lifecycle: https://github.com/buildpacks/lifecycle/blob/8821424a7dbe49630a70f57ccc53ef109b24d428/buildpack/build.go#L361.

Reply:
Could we file a bug/feature request for this on the lifecycle?

Reply:
Actually, looking through the spec, do we expect this field to be populated for the BOM?

Reply:
> Actually, looking through the spec, do we expect this field to be populated for the BOM?
[Looks like a no](https://github.com/buildpacks/spec/blob/main/platform.md#iobuildpacksbuildmetadata-json)

Reply:
Since this functionality won't be observed, choosing to retract this code.